lnrpc/rpc.proto

```diff
@@ -2295,6 +2295,35 @@ message Payment {
 
     ///  The fee paid for this payment in milli-satoshis
     int64 fee_msat = 12 [json_name = "fee_msat"];
+
+    /// The time in UNIX nanoseconds at which the payment was created.
+    int64 creation_time_ns = 13 [json_name = "creation_time_ns"];
+
+    /// The HTLCs made in attempt to settle the payment [EXPERIMENTAL].
+    repeated HTLCAttempt htlcs = 14 [json_name = "htlcs"];
+}
+
+message HTLCAttempt {
+        enum HTLCStatus {
+                IN_FLIGHT = 0;
+                SUCCEEDED = 1;
+                FAILED = 2;
+        }
+
+        /// The status of the HTLC.
+        HTLCStatus status = 1 [json_name = "status"];
+
+        /// The route taken by this HTLC.
+        repeated bytes path = 2 [json_name = "path"];
```

Comment:
A question is whether we should take the opportunity to expand the information that is possible to get from the path field. If we instead exposed the full "route" here, a caller could get useful information such as fees etc.

Reply:
not a bad idea, we could reuse the existing route serialization?

Reply:
SGTM

Reply:
done